.github/workflows/build-test-osx-m1.yaml

```diff
@@ -0,0 +1,91 @@
+name: build-test-osx-m1
+
+on:
+  workflow_call:
```

Comment:
Can we add a workflow_dispatch here as well just in case it's needed in the future?

Reply:
Done